omf-python/tests/test_pointset.py

```diff
@@ -29,55 +28,3 @@ def test_should_return_expected_vertices(self) -> None:
         # Then
         expected_vertices = [[-1.0, -1.0, 0.0], [1.0, -1.0, 0.0], [1.0, 1.0, 0.0], [-1.0, 1.0, 0.0], [0.0, 0.0, 1.0]]
         self.assertEqual(vertices, expected_vertices)
```

Comment:
Does this get replaced by another test?

Reply:
No, I don't think so, this is testing the vertices on the geometry of the element, where as the attribute test will test a VectorArray, rather than a VetexArray

Reply:
Sorry I mean to the tests you removed below: `test_should_return_categories_attribute()` and `test_should_return_categories_attribute_data()`

Reply:
Oh right, yep they are tested for in the `omf-python/tests/test_category_attribute.py`